packages/flutter/lib/src/widgets/autofill.dart

```diff
@@ -0,0 +1,196 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'framework.dart';
+
+export 'package:flutter/services.dart' show AutofillHints;
+
+/// An [AutofillScope] widget that groups [AutofillClient]s together.
+/// [AutofillClient]s within the same [AutofillScope] will be autofilled together.
```

Comment:
Need a line break here. And I'd add "must be built together", like:

...the same [AutofillScope] must be built together and will be autofilled together